src/services/invitation/utils.ts

```diff
@@ -0,0 +1,36 @@
+import * as Papa from 'papaparse';
+
+import { BusboyFileStream } from '@fastify/busboy';
+
+import { PermissionLevel } from '@graasp/sdk';
+
+export type CSVInvite = {
+  email: string;
+  name?: string;
+  group_name?: string;
+  permission?: PermissionLevel;
+};
+
+export const getCSV = (
+  stream: BusboyFileStream,
+): Promise<{ rows: CSVInvite[]; header: string[] }> => {
+  return new Promise((resolve, reject) => {
+    Papa.parse(stream, {
+      header: true,
+      dynamicTyping: true,
+      complete(results, file) {
+        resolve({
+          rows: results.data as CSVInvite[],
+          header: results.meta.fields,
+        });
+      },
+      error(err, file) {
+        reject(err);
+      },
+    });
+  });
+};
+
+export const regexGen = (firstLvlPath: string) => {
+  return RegExp(`${firstLvlPath}\.[a-zA-Z0-9_]+$`);
+};
```

Comment:
Is this supposed to check whether an item is a descendant of the item passed as argument ? Make this purpose clear in the function name please.